dotcom-rendering/src/web/layouts/FrontLayout.tsx

```diff
@@ -9,9 +9,8 @@ import { Nav } from '../components/Nav/Nav';
 import { Island } from '../components/Island';
 import { MostViewedFooterLayout } from '../components/MostViewedFooterLayout';
 import { decidePalette } from '../lib/decidePalette';
-import { ContainerLayout } from '../components/ContainerLayout';
 import { Header } from '../components/Header';
-import { DynamicFast } from '../components/DynamicFast';
+import { renderFrontContainer } from '../lib/renderContainer';
```

Comment:
Can this lib import be grouped with other libs?

Reply:
I don't think so, since there's no single export file from the `/lib` directory.

Reply:
Nevermind I think I know what you mean haha!

Reply:
Sorry, I meant as in the line ordering. Probably one for an ESLint config to sort imports alphabetically!